drivers/sensor/hc_sr04/hc_sr04.h

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2024 Adrien Leravat
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_DRIVERS_SENSOR_HCSR04_HCSR04_H_
+#define ZEPHYR_DRIVERS_SENSOR_HCSR04_HCSR04_H_
+
+#include <zephyr/device.h>
+#include <zephyr/drivers/gpio.h>
+
+#define HC_SR04_MM_PER_MS     171
+
+struct hcsr04_data {
+	const struct device *dev;
+	struct gpio_callback gpio_cb;
+	struct k_sem sem;
+	uint32_t start_cycles;
+	atomic_t echo_high_cycles;
+};
+
+struct hcsr04_config {
+	struct gpio_dt_spec trigger_gpios;
+	struct gpio_dt_spec echo_gpios;
+};
```

Comment:
There's not enough here to justify having a separate private include header, please move these to the .c file.

Reply:
Done.